app/main.py

```diff
@@ -1,6 +1,17 @@
 from typing import Callable
+from typing import Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    result_dict = {}
+
+    def wrapper(*args, **kwargs) -> Any:
+        if args in result_dict:
+            print("Getting from cache")
```

Comment:
* use `functools.wraps`

Why you check only args?
What if function takes kwargs?
You must use args + kwargs as a key